placementApp/urls.py

```diff
@@ -3,7 +3,8 @@
 from . import views
 
 router = routers.DefaultRouter()
-router.register("students", views.StudentViewSet)
+router.register("student_list", views.ListStudentViewSet)
```

Comment:
Please use paths if you are making an individual url. routers are usually used for implementing multiple request type endpoint together (like a view having retrieve, update, delete functionality).

Reply:
> Please use paths

I can use **as_view** and **path()** for the endpoint as per [the documentation](https://www.django-rest-framework.org/tutorial/6-viewsets-and-routers/#binding-viewsets-to-urls-explicitly), but
>  routers are usually used for implementing multiple request type endpoint together

Currently the endpoint is doing retrieve and update operations (Get, Put,patch).

Is the switch required?

Reply:
I suggest you to use paths. Routers are usually used when a single endpoint has all requests like "GET", "POST", "PATCH", "PUT", "DELETE".

Reply:
Have made the changes. Thanks for the clarification.

Reply:
Have made the changes, [here](https://github.com/kanishkshah/placement-portal-web/commit/787558933f3595a8a16c7c02b4ca9e3b8a940d25) are the latest commit changes.
Since Vartak also edited views and urls, got a merge conflict and resolved it on my local machine.
**However my View depended on a bit of Saakshi's work, that conflict remains unresolved** , hence I cannot test the functionality of my endpoints at the moment.
I should hold off from pushing the merged code and push it once I know it is ready right?

Reply:
Just resolve the conflicts for your own urls and check the views. Interdependent part can be done later.